src/ontology/mondo-ingest.Makefile

```diff
@@ -160,15 +160,20 @@ mappings: sssom $(ALL_MAPPINGS)
 #################
 # Utils #########
 #################
-# Documentation for this commands in this section is in: `docs/developer/ordo.md`
-
+# Documentation for `report-mapping-annotations` and `update-jinja-sparql-queries`: `docs/developer/ordo.md`
 report-mapping-annotations:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/report_mapping_annotations.py
 
 update-jinja-sparql-queries:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
+config/%_term_exclusions.txt: config/%_exclusions.tsv component-download-%.owl
+	python3 ../scripts/exclusion_term_expansion/exclusion_term_expansion.py \
+	--exclusions-path $(word 1,$^) \
+	--onto-path $(TMPDIR)/$@.owl \
```

Comment:
@matentzn 
## Summary
I think you had a bug in one of the recommended changes to this make command. But I fixed it.

## Full bugfix description
(a) `--onto-path $(TMPDIR)/$@.owl` vs (b) `--onto-path $(word 2,$^)` vs (c) (see below)

(b) is how I originally had things in my last update, but you recommended a change to (a). I think this is a mistake.
- (a) expands to: `--onto-path tmp/config/icd10cm_term_exclusions.txt.owl`
- (b) expands to: `--onto-path component-download-icd10cm.owl

I looked at `tmp` and it looks like the file that I need to be referencing is: `component-download-icd10cm.owl.owl`

Therefore, I think I fixed this bug by changing to (c): `--onto-path $(TMPDIR)/$(word 2,$^).owl`
...which expands to: `tmp/component-download-icd10cm.owl.owl`

Reply:
@matentzn Per requested reduction of FYI comments I'm going to mark this one resolved.